content/applications/hr/payroll/payslips.rst

```diff
@@ -28,159 +30,178 @@ from this dashboard.
-Days` and :guilabel:`Salary Computation` tabs.
+A confirmation pop-up window asks, `Are you sure you want to proceed?`. Click :guilabel:`OK` to
+confirm.
 
-To print the payslip, click the :guilabel:`Print` button. To cancel the payslip, click the
-:guilabel:`Cancel Payslip` button.
+The chatter is automatically updated to show the email sent to the employee, along with a PDF copy
+of the payslip.
 
-Once everything on the payslip form is correct, click the :guilabel:`Create Draft Entry` button to
-create the payslip. The chatter is automatically updated to show the email sent to the employee,
-along with a PDF copy of the payslip.
+To print the payslip, click the :guilabel:`Print` button. To cancel the payslip, click the
+:guilabel:`Cancel` button.
 
 .. image:: payslips/payslip-chatter.png
    :align: center
    :alt: The new payslip is emailed to the employee and the email appears in the chatter.
 
-Next, the payment must be sent to the employee. To do this, click the :guilabel:`Make Payment`
+Next, the payment must be sent to the employee. To do this, click the :guilabel:`Register Payment`
 button. Doing so reveals a pop-up form, in which the desired :guilabel:`Bank Journal` that the
 payment should be made against must be selected from a drop-down menu. Then, click the
 :guilabel:`Confirm` button to confirm the journal, and return to the payslip.
 
-.. image:: payslips/make-payment.png
-   :align: center
-   :alt: Click Make Payment to send the payment to the employee.
+Odoo automatically checks bank account information. If there is an error with the employee's listed
+bank account, an error appears in a pop-up window, stating, `The employee bank account is
+untrusted.` If this error appears, update the employee's bank account information on their
+:ref:`Employee Form <employees/private-info>`.
 
-If a payment needs to be cancelled or refunded, click the corresponding :guilabel:`Refund` or
-:guilabel:`Cancel Payslip` button, located at the top of the payslip form.
+If a payment needs to be canceled or refunded, click the corresponding :guilabel:`Cancel` or
+:guilabel:`Refund` button, located at the top-left of the screen.
 
 .. important::
    In order for a payslip to be paid, the employee *must* have a bank account entered in their
-   contact information. If there is no bank information, a payslip cannot be paid, and an error will
-   appear when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
-   the :guilabel:`Private Information` tab on the employee's card. Edit the employee card, and add
-   banking information, if it is missing.
+   contact information. If there is no bank information, a payslip cannot be paid, and an error
+   appears when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
+   the :ref:`Private Information <employees/private-info>` tab on the employee's card. Edit the
+   employee card, and add banking information if it is missing.
```

Comment:
I'm assuming this is why I continued to get that error...? not sure. If so, since I encountered that issue after click 'Register Payment' this admonition may be best if placed right after that step.

Reply:
I agree! That ight have been why- this has been moved up.